apps/email/js/cards/lst/default_vscroll_data.js

```diff
@@ -0,0 +1,31 @@
+// Default data used for the VScroll component, when data is not
+// loaded yet for display in the virtual scroll listing.
```

Comment:
enhancement request: maybe we could put in the comment an explanation of what an \u2583 is.  Or the actual literal character, depending on how much we were trying to avoid putting high characters in.